### ISSUE TITLE
.Net Core 3

### ISSUE THREAD

**Issue by amartens181** [MicrosoftDocs/windows-uwp#1841]
Is this possible in .net core 3 WPF apps? It doesn't seem to be.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 14e9484a-4921-2ee3-c93e-787a7b104f61
* Version Independent ID: 9e3db713-1558-7297-ef6d-53e5d87d4757
* Content: [Send a local toast notification from desktop C# apps - Windows UWP applications](https://docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/send-local-toast-desktop)
* Content Source: [windows-apps-src/design/shell/tiles-and-notifications/send-local-toast-desktop.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/shell/tiles-and-notifications/send-local-toast-desktop.md)
* Product: **uwp**
* Technology: **design**
* GitHub Login: @mijacobs
* Microsoft Alias: **mijacobs**

**Comment by andrewleader** [comment]
Hey @amartens181, it is possible with .NET Core 3.0, we're working on updating the helper library for .NET Core 3.0 along with publishing the helper library as part of the Windows Community Toolkit.

Here's the latest branch we've been working on: https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/nmetulev/desktopnotifications/Microsoft.Toolkit.Uwp.Notifications/DesktopNotificationManager

You might be able to use that code and start sooner, as it'll still be some time till we publish the package and update the documentation.

**Comment by hickeys** [comment]
closing this issue as there is no documentation change required

**Comment by hickeys** [comment]
Mistakenly closed this issue, reopening since Andrew is already working on it.

**Comment by szensk** [comment]
>Here's the latest branch we've been working on: https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/nmetulev/desktopnotifications/Microsoft.Toolkit.Uwp.Notifications/DesktopNotificationManager

This link is dead. How do you replace [RegistrationServices](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.registrationservices?view=netframework-4.8)? It's not part of .NET Core apparently.

**Comment by andrewleader** [comment]
Hey everyone, we've updated the code to work with .NET Core 3: https://github.com/WindowsNotifications/desktop-toasts/blob/master/CS/DesktopToastsApp/DesktopNotificationManagerCompat.cs

We're also in the process of adding this code to the Community Toolkit, but until then you'll have to copy/paste this file locally as the documentation says.

Closing since link in documentation has also been updated to the code that works with .NET Core 3. Let us know if you have any problems!
